trezor-keyring-mv2.js

```diff
@@ -0,0 +1,49 @@
+const { default: TrezorConnect, DEVICE_EVENT } = require('trezor-connect');
+const {
+  TrezorKeyring,
+  KEYRING_TYPE,
+  TREZOR_CONNECT_MANIFEST,
+} = require('./trezor-keyring');
+
+class TrezorKeyringMv2 extends TrezorKeyring {
```

Comment:
This class doesn't appear to be the exact same as the previous TrezorKeyring class. For instance:

* The initialization logic that was in the constructor has been moved to `init`
* `dispose` now returns a promise

Is that intentional in this PR? Those would be breaking changes if so, in which case we no longer have to have `index.js` expose a default export.

Reply:
Adding the `init` method and making the `dispose` method return a promise in case that other implementations of the bridge logic require it are intentional changes. I've added a PR for the KeyringController to take the `init` method into account, as other keyring implementations might need it as well.

What I'm not too sure about is if I should make them completely optional by not mentioning them at all in the base class, return an error or return an empty promise.

I'm starting to lean towards not defining them at all in the base class, then have a separate interface in the exported types in case they have to be added so that whoever implements them has a function signature, but would like to discuss it further.